pkg/logutil/logutil.go

```diff
@@ -22,10 +22,22 @@ const (
 	oracleIDLoggerKey     = "oracleID"
 	donIDLoggerKey        = "donID"
 	configDigestLoggerKey = "configDigest"
+	ocrPhaseLoggerKey     = "ocrPhase"
 
 	// ocrSeqNrKey refers to the OCR sequence number.
 	// Linter complains if this is a plain "string" type.
 	ocrSeqNrKey = contextKey(ocrSeqNrLoggerKey)
+	ocrPhaseKey = contextKey(ocrPhaseLoggerKey)
+)
+
+// OCR phase values, all loggers should use these to standardize output.
+const (
+	PhaseQuery          = "qry"
+	PhaseObservation    = "obs"
+	PhaseOutcome        = "otcm"
+	PhaseReports        = "rprt"
+	PhaseShouldAccept   = "sacc"
+	PhaseShouldTransmit = "strn"
 )
```

Comment:
Not sure about the shorthand. Consider a type with long/short forms.

Reply:
Logic behind the shorthand was to not contribute too much to log payload sizes